bin/setup-policy-routes.sh

```diff
@@ -310,6 +310,34 @@ setup_interface() {
     echo $changes
 }
 
+# All instances of this process that may reconfigure networkd register
+# themselves as such. When exiting, they'll reload networkd only if
+# they're the registered process running.
+maybe_reload_networkd() {
+    rm -f /run/setup-policy-routes/$$
```

Comment:
Would using /dev/shm as the root for all these files better serve us by eliminating the entire class of bugs related to deadlocking across a reboot?